app/controllers/admin/custom_fields/custom_field_projects_controller.rb

```diff
@@ -110,11 +129,14 @@ def find_projects_to_activate_for_custom_field
       respond_with_turbo_streams
     end
   rescue ActiveRecord::RecordNotFound
-    update_flash_message_via_turbo_stream message: t(:notice_project_not_found), full: true, dismiss_scheme: :hide,
-                                          scheme: :danger
-    update_project_list_via_turbo_stream
+    respond_with_project_not_found_turbo_streams
+  end
 
-    respond_with_turbo_streams
+  def find_custom_field_project_to_destroy
+    @project = Project.find(params.to_unsafe_h[:custom_fields_project][:project_id])
```

Comment:
I am rather sure that you don't need the `to_unsafe_h` here. You could just write

```
@project = Project.find(params[:custom_fields_project][:project_id])
```

Reply:
Come to think of it, I don't understand why the project is fetched at all. Couldn't it also be written like this:

```
@custom_field_project = CustomFieldsProject.find_by!(custom_field: @custom_field, 
                                                     project: params[:custom_fields_project][:project_id])
```

That of course is only true if `@project` is really not used at all. But you'd safe a database request if it needn't be loaded.

Reply:
Great catch, missed that! 👍🏾